package.json

```diff
@@ -16,19 +16,19 @@
     "nodejs"
   ],
   "author": "Erik Toth <ertoth@paypal.com>",
-  "licenses": [
-    {
-      "type": "Apache 2.0",
-      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
-    }
-  ],
+  "license": "Apache-2.0",
   "dependencies": {
-    "core-util-is": "^1.0.0"
+    "core-util-is": "^1.0.0",
+    "debuglog": "^1.0.1",
+    "dustjs-helpers": "~1.7.0",
+    "dustjs-linkedin": "~2.7.1",
+    "freshy": "^1.0.0",
+    "path-is-absolute": "^1.0.0",
+    "verror": "^1.6.0"
   },
   "devDependencies": {
     "chai": "^1.5.0",
-    "dustjs-helpers": "~1.6.3",
-    "dustjs-linkedin": "~2.6.0",
+    "engine-munger": "^1.0.0-0",
```

Comment:
Nah, if it works, I think we're in good shape. I'd love to run tests against 3, 4, and 5 when it comes out though.